package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@primer/components",
-  "version": "14.3.1",
+  "version": "14.3.2",
```

Comment:
This is new as of today, but since there's been so much activity in this repo lately (yay! 🎉) I think we should start doing release tracking PRs again to avoid versioning errors with so many things being shipped! If you want to just revert this change and then change the base branch to `release-14.4.0` we should be good to go!

Reply:
Got it!